src/gateways/createOSWebLoader.ts

```diff
@@ -29,30 +29,28 @@ export default (prefix: string) => {
   const baseUrl = `${prefix}/v2/pages`;
   const toJson = (response: any) => response.json() as Promise<OSWebResponse>;
 
-  const firstRecord = (id: string) => (data: OSWebResponse) => {
-    if (!data.items[0]) {
-      throw new Error(`OSWeb record "${id}" not found`);
-    }
-    return data.items[0];
-  };
+  const firstRecord = (data: OSWebResponse) => data.items[0];
 
   const cache = new Map();
 
-  const cacheRecord = (record: OSWebBook) => {
+  const cacheRecord = (record?: OSWebBook) => {
+    if (!record) {
```

Comment:
we still want to cache the empty response, so that we don't try to query it again